librz/magic/magic.c

```diff
@@ -48,6 +48,14 @@ RZ_API const char *rz_magic_error(RzMagic *m) {
 RZ_API void rz_magic_setflags(RzMagic *m, int f) {
 	magic_setflags(m, f);
 }
+RZ_API bool rz_magic_load_buffer(RzMagic *m, const char *f) {
+	if (*magicdata == '#') {
```

Comment:
Should be f.
The tests are passing but the build will break with USE_LIB_MAGIC

Reply:
Please send a PR